### DISCUSSION TITLE
Value Mapping to rename values using RegEx

### DISCUSSION THREAD

**Discussion by HorselessName** [grafana/grafana#63777] [Dashboards, Panels & Visualizations]
Hello there...

I applied the following transformations in my Panel Visualization:
Labels to fields, Reduce and Organize Fields.

![image](https://user-images.githubusercontent.com/20173635/221371102-d2304b9c-9a98-4c01-8319-6a4f5d8dfdcd.png)

Now I am trying to rename my services using a value mapping, but the RegEx is not working.
The RegEx I am using is, in Value Mapping:  `(^Status do Serviço\s*)` → ABC

![image](https://user-images.githubusercontent.com/20173635/221371154-f280f5d6-4506-4375-a444-53f697a9e52c.png)

If I change the RegEx to `.*` it works. On RegEx101 the RegEx above also is valid...
What am I not getting?

**Comment by HorselessName** [comment]
![image](https://user-images.githubusercontent.com/20173635/221371285-c5e25e2e-e968-4d3f-9629-97dde23dcbc7.png)

**Reply by HorselessName** [reply]
This is the working example using RegEx `.*` that renames the entire thing instead of just removing the "Status of Service" part.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82840. Feel free to continue the discussion around this there. Thank you!
